Automattic/wp-calypso
# Pattern Assembler - Track category_slug in calypso_signup_pattern_assembler_pattern_final_select

Related to https://github.com/Automattic/wp-calypso/issues/73363

## Proposed Changes

* Add a new pattern `category_slug` into the JSON data 
* Pass `category_slug` as `category` property in the event `calypso_signup_pattern_assembler_pattern_final_select`

## Testing Instructions



* Using Tracks Vigilante extension
* Access `/start?siteSlug=[ your site slug ]`
* Select the goal "Write and publish"
* Continue until the Design picker and scroll down until the CTA "Design your own"
* Click "Start designing"
* Add sections and continue
* Verify that the tracks event `calypso_signup_pattern_assembler_pattern_final_select` contains a category slug in the prop `category`

<img width="793" alt="Screenshot 2566-02-15 at 16 24 25" src="https://user-images.githubusercontent.com/1881481/218991054-39592b7a-cd62-4c87-a60c-bcb0f562a058.png">

## Pre-merge Checklist
